src/components/Card/index.js

```diff
@@ -0,0 +1,21 @@
+import "./style.css";
+
+const CardComponent = ({ member }) => {
+  return (
+    <div className="card">
+      <img className="member-img" src={member.img}></img>
```

Comment:
img elements must have an alt prop, either with meaningful text, or an empty string for decorative images. 
[jsx-a11y/alt-text](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/tree/HEAD/docs/rules/alt-text.md)